rerun_cpp/src/rerun/archetypes/bar_chart_ext.cpp

```diff
@@ -0,0 +1,126 @@
+#include "bar_chart.hpp"
+
+// #define EDIT_EXTENSION
+
+namespace rerun {
+    namespace archetypes {
+
+#ifdef EDIT_EXTENSION
+        // [CODEGEN COPY TO HEADER START]
+
+        inline BarChart(rerun::datatypes::TensorBuffer buffer) {
+            this->values = rerun::components::TensorData::one_dim(std::move(buffer));
+        }
+
+        // --------------------------------------------------------------------
+        // Implicit constructors:
+
+        /// Construct a `BarChart` from a `std::vector<uint8_t>`.
+        inline BarChart(std::vector<uint8_t> u8)
```

Comment:
Let's not use `inline` inside of class declarations at any point: We haven't done so so far and it's completely redundant. See https://isocpp.org/wiki/faq/inline-functions#inline-member-fns-more

Reply:
applies to almost _every_ function in this PR 😄